app/views/users/_form.html.erb

```diff
@@ -46,8 +45,11 @@
     <label>Email:</label>
     <%= text_field_tag :email, current_user.email %><br>
 
+    <label>Please enter password to confirm:</label>
+    <%=text_field_tag :password, current_user.password %>
+
   <% end %>
 
   <%= submit_tag button_text %>
-
-<% end %>
+  <%= link_to "Edit Password", "/profile/edit/password" %>
+  <% end %>
```

Comment:
You added this button on the '/profile/edit' page but I think the user story wants it on the '/profile' page. 

If you change this remember to change your test as well.